providers/route53/route53Provider.go

```diff
@@ -13,27 +14,41 @@ import (
 	"github.com/aws/aws-sdk-go/aws/credentials"
 	"github.com/aws/aws-sdk-go/aws/session"
 	r53 "github.com/aws/aws-sdk-go/service/route53"
+	r53d "github.com/aws/aws-sdk-go/service/route53domains"
+	"github.com/aws/aws-sdk-go/private/waiter"
```

Comment:
Is it kosher to use "private" packages like this? My gut says no, but waiters are super-convenient (related to another question below).